test/fixtures/box/label.js

```diff
@@ -0,0 +1,102 @@
+module.exports = {
```

Comment:
Looking at the fixture, I think the label should be clipped inside the border (not draw halfway over it)